docs/modules/for-developers/pages/architecture.adoc

```diff
@@ -47,3 +47,7 @@ and crash the backend application.
 
 Because a single Docker host does not scale very well we plan to support other container orchestration systems in the
 future.
+
+== Data Structure
+The following schema is used in the underlying database:
```

Comment:
The problem is the graphics will be outdated as soon as we change anything in the schema. But, I agree that it will be useful for understanding our DB model. Maybe we could add a notice that the basic ER model is correct but individual columns might be outdated.
@erikhofer wdyt?

Reply:
Yeah, we should only add diagrams that we can keep up to date. I also think the ER model would be more useful than the logical database schema. We should also think about abstraction levels, a flattened out view of everything will become confusing with a few more entities.
But we could merge this now and revisit it fpr the arc42 documentation 🤔

Reply:
@fwoelk could you please add a sentence that states this DB schema might be outdated?